CONTRIBUTING.md

```diff
@@ -27,3 +27,29 @@ for details, such as:
 * Development setup and testing your changes
 * Submitting a pull request and finding a reviewer
 
+To build and install the whole project from the source distribution,
+you need additional tools installed in your system including:
+
+* JDK 8 or later
+* Python 3
+* pip
+* setuptools
+* Go 1.12 or later
+* virtualenv
+* tox
+* Docker
+
+To install these in a Debian-based distribution:
+
+```
+sudo apt-get install \
+    openjdk-8-jdk \
+    python-setuptools \
+    python-pip \
+    virtualenv \
+    tox \
+    docker-ce
```

Comment:
Where is Go 1.12?

Reply:
Go can't be installed by apt-get. Added pointer to instructions.

Reply:
Thanks.